packages/trader/src/RangeTrader.ts

```diff
@@ -138,9 +138,12 @@ export class RangeTrader {
         ...commonLogObject,
         error: tradeExecutionTransaction
       });
-      return;
+      throw tradeExecutionTransaction;
```

Comment:
if there is an error at this point it MUST throw, not return silently.

Reply:
Out of curiosity, why?

Reply:
it can revert in the dsproxy call and then just continue on the execution loop without producing any logs. obviously that's not good :)